frontend/src/components/CreateGoal/ObjectiveForm.js

```diff
@@ -164,6 +176,8 @@ ObjectiveForm.propTypes = {
       label: PropTypes.string,
       value: PropTypes.number,
     })),
+    // eslint-disable-next-line react/forbid-prop-types
+    files: PropTypes.arrayOf(PropTypes.object),
```

Comment:
same as above, we know the keys we expect the object to have

Reply:
Updated with actual shape.